app/services/flow/flows/citizen_start.rb

```diff
@@ -21,7 +21,14 @@ class CitizenStart < FlowSteps
         true_layer: {
           path: ->(_) { omniauth_login_start_path(:true_layer) }
         },
+        # :nocov:
```

Comment:
I'm not keen on adding :nocov: can we work out why this isn't being hit?
Maybe adding a request spec?